### DISCUSSION TITLE
next.js can't load page if polyfill.min.js is not available, ios safari

### DISCUSSION THREAD

**Discussion by linzack** [vercel/next.js#43222] [Help] [answered]
Hi, 
in my environment can't access polyfill.io, but next.js will require polyfill.min.js to load page on ios safari.
on dev tool console, the cause is this error:
`TypeError: undefined is not a constructor (evaluating 'new Intl.RelativeTimeFormat("en-US",{numeric:"auto"})')`
and the web page do not load, only show this error message:
`A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred`

I think if next.js have error handling to check polyfill is not available, and still load web page will be great. thank you!

**Comment by icyJoseph** [comment, marked as answer]
I don't think they polyfill for the Internationalization API:

- https://github.com/vercel/next.js/blob/canary/packages/next-polyfill-nomodule/src/index.js
- https://nextjs.org/docs/basic-features/supported-browsers-features#polyfills

So you need to do that work yourself. Are you sure it is next.js that's trying to reach for polyfill.min.js? Got a screenshot? Also what Safari version is this targeting?

**Comment by linzack** [comment]
Hi, 
after some investigation, I find the problem may not related to next.js.

I am not actually a web dev, just a user. 
What I can tell is that, if I control the only variable here, the availability of polyfill.min.js, 
the absence of polyfill will eventually cause client-side exception of next.js, but maybe there is other package to cause this problem.
I realized that after compare the pass / fail screenshot.
Thank you!
